cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,386 @@
+/*
+Copyright 2023 Suguru Hirahara
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+import { SettingLevel } from "../../../src/settings/SettingLevel";
+import { Layout } from "../../../src/settings/enums/Layout";
+
+describe("Audio player", () => {
+    let homeserver: HomeserverInstance;
+    const TEST_USER = "Hanako";
+
+    const percyCSS =
+        // FIXME: hide mx_SeekBar because flaky - see https://github.com/vector-im/element-web/issues/24898
+        ".mx_SeekBar, " +
+        // Exclude various components from the snapshot, for consistency
+        ".mx_JumpToBottomButton, " +
+        ".mx_MessageTimestamp, .mx_RoomView_myReadMarker { visibility: hidden !important; }";
+
+    const uploadFile = (file: string) => {
+        // Upload a file from the message composer
+        cy.get(".mx_MessageComposer_actions input[type='file']").selectFile(file, { force: true });
+
+        cy.get(".mx_Dialog").within(() => {
+            // Find and click primary "Upload" button
+            cy.get("[data-testid='dialog-primary-button']").findButton("Upload").click();
+        });
+
+        // Wait until the file is sent
+        cy.get(".mx_RoomView_statusArea_expanded").should("not.exist");
+        cy.get(".mx_EventTile.mx_EventTile_last .mx_EventTile_receiptSent").should("exist");
+    };
+
+    /**
+     * Take snapshots of mx_EventTile_last on each layout, outputting log for reference/debugging.
+     * @param detail The Percy snapshot name. Used for outputting logs too.
+     */
+    const takeSnapshots = (detail: string, monospace = false) => {
```

Comment:
please could you add `monospace` to the documentation for this function?

Reply:
Right, I've missed it. Thanks!